packaging/makeself/makeself-header.sh

```diff
@@ -1,4 +1,5 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
If these files were not written by us, them maybe we should just disable shellcheck on them?